server/methods/accounts/accounts.js

```diff
@@ -580,7 +580,7 @@ export function inviteShopOwner(options) {
 
   Reaction.Email.send({
     to: email,
-    from: `${_.get(dataForEmail, "primaryShop.name")} <${_.get(dataForEmail, "primaryShop.emails[0].address")}>`,
+    from: `${dataForEmail.primaryShop.name} <${dataForEmail.primaryShop.emails[0].address}>`,
```

Comment:
One thing that `_.get` does here is guard against errors, so we need to be 100 percent certain that primaryShop.name and primaryShop.emails[0].address always exist in this scenario, if there's a chance that's not the case, we should probably build `const`s that guard for us before calling send.